angular-rails-app/app/assets/javascripts/property/property.service.js

```diff
@@ -9,6 +9,30 @@ angular
   this.getProperty = function (id) {
     return $http.get(API_URL + '/properties/' + id)
   };
+
+  this.getComplaint311s = function (id, pageNumber){
+    return $http.get(API_URL + '/complaint311s?' + 'property_id=' + id + '&page=' + pageNumber)
+  }
+
+  this.getDobPermits = function (id, pageNumber){
+    return $http.get(API_URL + '/dob_permits?' + 'property_id=' + id + '&page=' + pageNumber)
+  }
+
+  this.getDobViolations = function (id, pageNumber){
+    return $http.get(API_URL + '/dob_violations?' + 'property_id=' + id + '&page=' + pageNumber)
+  }
+
+  this.getHpdComplaints = function (id, pageNumber){
+    return $http.get(API_URL + '/hpd_complaints?' + 'property_id=' + id + '&page=' + pageNumber)
+  }
+
+  this.getHpdViolations = function (id, pageNumber){
+    return $http.get(API_URL + '/hpd_violations?' + 'property_id=' + id + '&page=' + pageNumber)
+  }
+
+  this.getLitigations = function (id, pageNumber){
+    return $http.get(API_URL + '/litigations?' + 'property_id=' + id + '&page=' + pageNumber)
+  }
```

Comment:
Yep, I agree with this, and it's a bit more than a nit. The more times the code is duplicated, the more likely there is to be a copy-paste error. 

The sense I'm getting is that all these related data types should more or less work the same way. If we want to improve of change functionality across all these types later, we have to do it in {# types} places. Unless there's a strong intention that I don't know about for these types to act differently, I think we should combine the code.